src/services/info/notice/modules/list-item/ListItem.vue

```diff
@@ -53,6 +55,7 @@ interface Props {
     isNew: boolean;
     isPinned: boolean;
     postDirection: 'prev' | 'next' | undefined;
+    post?: NoticePostModel;
```

Comment:
post 는 무조건 있는것 아닌가용??
```suggestion
    post: NoticePostModel;
```

Reply:
post list를 가져오기 전에는 undefined라고 생각해서 ?를 추가하였습니다

Reply:
그 때에는 ListItem 컴포넌트가 생성되지 않은 상태 아닌가용?

Reply:
아! 포문 안돌지.. 감사합니다!